_posts/2015-09-18-epages-you-2015.md

```diff
@@ -0,0 +1,103 @@
+This sounds really tough.
+But it is not.
+It is incredibly fun, although it is hard work, too and involves long hours.
+
+Main goal of the event:
+
+* get to know each other better
+* unite colleagues from all company locations
+* exchange ideas among colleagues, departments and countries.
+
+# Event task force
+
+Already a long time before the actual date of the YOU, an event task force started their activities to prepare the ground for two awesome days.
+To make it a big surprise for everyone, the agenda was kept a secret for a long time.
+But soon the time had come to position some teasers and involve the employees.
+
+# Prepare for action
+
+Every team was asked to prepare a 1-minute video, that presents their departments.
+To make it short: that was a hell of a lot of fun!
+We also had some restrictions for the team presentations itself: hold a 12-minutes presentation and prepare for a 12-minutes Q&A session.
+That was definitely a challenge and required time for a careful preparation.
+
+{% image blog-you-prepare.jpg 100% %}{% endimage %}
+
+# Patience required
+
+There were rumors that we would go for an outdoor activity.
+Soon you heard colleagues whisper and chatter on the corridors, what that might be.
+Climbing? Geocaching? City walk?
+No chance to find that out.
+Staying patient was required.
+
+# Kick-off at the beach
+
+{% image blog-you-beachclub.jpg 100% %}{% endimage %}
+
+Finally the YOU started in a beach club above the city roofs of Hamburg.
+We enjoyed a beautiful view across the city, along with food and drinks, a relaxed atmosphere and lots of fun.
+Incidentally, we also had the chance to observe a beautiful sundown!
+
+# Opening YO(U)
+
+The next day we all convened at the Hamburg University to spend the first day watching funny and awesome team videos and listening to well-prepared team presentations.
+
+{% image blog-you-presentation.jpg 100% %}{% endimage %}
+
+Exciting: which team was going to present was drawn right before the presentation, so every team had to be prepared at all times!
+
+# Dragon boat race
+
+In the afternoon the secret was finally lifted what would be our outdoor activity: we went to a small dock at a branch of the Alster to spend the afternoon with dragon boat races.
+
+{% image blog-you-dragonboat.jpg 100% %}{% endimage %}
+
+This outdoor activity was simply spectacular!
+There was so much enthusiasm and ambition amongst the colleagues learning the skills, performing the strokes in sync, being one effective entity to win the race - with a good battle cry, of course.
+
+# Being rewarded
+
+{% image blog-you-reward.jpg 100% %}{% endimage %}
+
+We finished the day chilling out at a restaurant with cocktails and burgers.
+The composition of the tables was chosen at random.
+Everyone again had to be out of their comfort zones to get to know new colleagues from other locations.
+
+# Prepare for the coming day
+
+Just one (short) night to recover from the current events and jump right into the next.
```

Comment:
Just one (short) night to recover from the current events and then it was time to jump right into the next.